### DISCUSSION TITLE
Ts .... for G4HPolycone

### DISCUSSION THREAD

**Discussion by bcmohamed** [OpenTOPAS/OpenTOPAS#95] [Feature-Request] [answered]
Hi
I am wondering what is the extension file in topas corresponds to G4HPolycone ?
Thank you

**Comment by tmasilela** [comment, marked as answer]
Greetings,

Please take a look at [this](https://opentopas.readthedocs.io/en/latest/examples-docs/Basic/ShapeTestWithAllParameters.html) example parameter file, along with the corresponding page about generic components on our [documentation](https://opentopas.readthedocs.io/en/latest/parameters/geometry/generic.html).

All the best,

Thongchai
